src/Uno.UI/UI/Xaml/Controls/TextBox/TextBoxView.wasm.cs

```diff
@@ -62,6 +58,7 @@ public TextBoxView(TextBox textBox, bool isMultiline)
 			}
 
 			SetAttribute("tabindex", "0");
+			UpdateContextMenuEnabling();
```

Comment:
Could the default value be the right one, so we avoid invoking this during the construction?